redux/package.json

```diff
@@ -1,8 +1,9 @@
 {
+  "name": "flux-comparison-redux",
```

Comment:
I added a name here because NPM thought I'm trying to install `redux` inside Redux itself.